modules/backend/controllers/Auth.php

```diff
@@ -48,6 +50,9 @@ public function index()
      */
     public function signin()
     {
+        // Add HTTP Header to Clear Cache and Cookies before Signin, see github issue: #3707
+        header("Cache-Control: no-cache, no-store, must-revalidate");
```

Comment:
We can't have headers being sent manually from October, everything must tie into the global response object that Laravel deals with. A middleware would be a better option here, it would also resolve the double headers.